packages/integration_tests/tests/actions/test_openshift_pod_restart.py

```diff
@@ -0,0 +1,141 @@
+import random
+from time import sleep
+
+from automated_actions_client import AuthenticatedClient
+from automated_actions_client.api.v1 import action_detail, openshift_workload_restart
+from automated_actions_client.models.action_schema_out import ActionSchemaOut
+from automated_actions_client.models.action_status import ActionStatus
+from automated_actions_client.models.openshift_workload_restart_kind import (
+    OpenshiftWorkloadRestartKind,
+)
+from kubernetes.dynamic.resource import ResourceInstance
+from pydantic import BaseModel
+
+from tests.conftest import Config
+from tests.utils import openshift_client
```

Comment:
Please avoid the  `test.utils` package and use the `pytest.fixture` feature instead.

Reply:
fixed in 0ec09236a98bdae2eabce91e08211769d5fd21a5